### ISSUE TITLE
Image Rotation

### ISSUE THREAD

**Issue by sweatapp** [jdamcd/android-crop#176]
I am getting an image rotation problem like many others are as well, what about just adding a rotate button to solve this problem. Just allow the user to rotate the image other than relying on the orientation in the file since it seems as if Samsung has dropped the ball on this issue.

**Comment by popboyking** [comment]
No,my problem is just me click the home or back button , then the screen is on the launcher.At this time ,back to the game ,the game was come back to the first page is not in palying state.



522295808@qq.com
 
From: Carson Aberle
Date: 2015-11-03 15:39
To: jdamcd/android-crop
Subject: [android-crop] Image Rotation (#176)
I am getting an image rotation problem like many others are as well, what about just adding a rotate button to solve this problem. Just allow the user to rotate the image other than relying on the orientation in the file since it seems as if Samsung has dropped the ball on this issue.

**Comment by davidgoli** [comment]
Or at the VERY least expose the CropUtils class. These are generally useful utilities that should be public, not package-level visibility.

**Comment by developer--** [comment]
how can I rotate image, but not rotate cropping tool. is it capable with your library?

**Comment by Jai-GAY** [comment]
ya, i notice this issue if i hold the phone in portrait, (height > width), the resultView will be rotated.

resultView.setImageURI(Crop.getOutput(result));

**Comment by tpietsc1** [comment]
This is related to CropUtil.copyExifRotation. This method is not writing Exif data since it's using android.media.ExifInterface.saveAttributes which is currently just a stub. Probably working someday - for the time being one could rotate the saved Bitmap or use another Exif lib.

**Comment by Kudaybergen** [comment]
How, how can I rotate it in crop mod?

**Comment by matte5031** [comment]
@sweatapp did u find any solution to thid?, thanks

**Comment by Nikaoto** [comment]
Getting the same problem here with a Sony android device, any fixes yet?
